includes/class-wc-checkout.php

```diff
@@ -685,28 +685,37 @@ public function get_posted_data() {
 				continue;
 			}
 
+			$form_was_shown = isset( $_REQUEST['woocommerce-process-checkout-nonce'] );
```

Comment:
Minor things: we could move outside of the foreach loop (no need to evaluate each iteration) and given the context we could be specific about the source (switch to `$_POST`).

Reply:
done f20e8ab